Gemfile.local

```diff
@@ -0,0 +1,7 @@
+# Want to extend Zammad with additional gems?
```

Comment:
I had to add this file to get Dependabot to run properly (it sees the `eval_gemfile` in the main Gemfile and tries to fetch the associated file, not realising there's a conditional that means it's OK not to). Hopefully having an empty template file like this one is fine, but if you'd like to use Dependabot and aren't happy with that then I can try to update Dependabot's file fetching logic.